bb-bootstrap-cards-module/bb-bootstrap-cards-module.php

```diff
@@ -41,6 +46,51 @@ public function update($settings)
         }
         return $settings;
     }
+
+    /**
+     * @method get_data
+     */
+    public function get_data()
+    {
+        if(!$this->data) {
+
+            // Photo source is set to "library".
+            if(is_object($this->settings->photo)) {
+                $this->data = $this->settings->photo;
+            }
+            else {
+                $this->data = FLBuilderPhoto::get_attachment_data($this->settings->photo);
+            }
+
+            // Data object is empty, use the settings cache.
+            if(!$this->data && isset($this->settings->data)) {
+                $this->data = $this->settings->data;
+            }
+        }
+
+        return $this->data;
+    }
+
+    /**
+     * @method get_alt
+     */
+    public function get_alt()
+    {
+        $photo = $this->get_data();
+
+        if(!empty($photo->alt)) {
+            return htmlspecialchars($photo->alt);
+        }
+        else if(!empty($photo->description)) {
+            return htmlspecialchars($photo->description);
+        }
+        else if(!empty($photo->caption)) {
+            return htmlspecialchars($photo->caption);
+        }
+        else if(!empty($photo->title)) {
+            return htmlspecialchars($photo->title);
+        }
+    }
```

Comment:
If user did not add $photo->alt, $photo->description, $photo->title any of them then alt attribute might go blank.